### PR TITLE
Add custom config variables for window resizing and shaders

### PR DESCRIPTION
**Pull request by jhamrick**
Shaders and window resizing don't seem to work very well currently on OSX, so it would be great to have the option to disable those things in Viewer. I've added two custom config variables to Config.prc: `viewer-use-shaders` and `viewer-resize-window`. These default to true, so it shouldn't change the current behavior unless they're overridden.

### CODE REVIEW

File: scenesim/display/viewer.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from libpanda import AntialiasAttrib, BitMask32, PerspectiveLens, Point3, Vec3
 from numpy import array
 from panda3d.bullet import BulletRigidBodyNode
-from panda3d.core import WindowProperties
+from panda3d.core import WindowProperties, ConfigVariableBool
```
Comment (pbattaglia) [inline scenesim/display/viewer.py]: alphabetize

File: scenesim/display/viewer.py [hunk 2]
```diff
@@ -26,7 +26,11 @@ class Viewer(ShowBase, object):
     def __init__(self):
         # ShowBase init.
         ShowBase.__init__(self)
-        self.win_size = (800, 800)
+
+        resize_window = ConfigVariableBool('viewer-resize-window', '#t')
+        print resize_window.getValue()
```
Comment (pbattaglia) [inline scenesim/display/viewer.py]: py 3 compatability: 
print(resize_window.getValue())

But actually, don't litter prints - I'd say kill it

**Comment by pbattaglia** [comment]
I'll merge the PR with those changes.

File: scenesim/display/viewer.py (hunk 2 above)
Comment (jhamrick) [inline scenesim/display/viewer.py]: Oh, oops, yeah -- I didn't mean to leave that in there. I'll push a new version.

**Comment by jhamrick** [comment]
Ok, updated the PR with your comments.

**Pull request merged**